lib/hooks/use-user-info.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useMemo, useState } from 'react';
+import { Member, useEcomApi } from '../ecom';
+
+export function useUserInfo() {
+    const api = useEcomApi();
+
+    const [user, setUser] = useState<Member>();
+
+    const isLoggedIn = useMemo(() => api.isLoggedIn(), [api]);
+
+    useEffect(() => {
+        if (api.isLoggedIn()) {
```

Comment:
`isLoggedIn`? (constant above)

I'll aks again if you'll ignore it again 😄

Reply:
didn't get what you mean previously😀